dist/build.gradle

```diff
@@ -128,3 +137,12 @@ task('distZip', type: Zip) {
 distribution {
     dependsOn(distZip)
 }
+
+// Add a task in the root project that collects all the dependencyReport data for each project
+// Concatenates the dependencies CSV files into a single file
+task generateDependenciesReport(type: ConcatFilesTask) {
+    dependsOn rootProject.allprojects.collect { it.tasks.withType(DependenciesInfoTask) }
+    files = fileTree(dir: project.rootDir,  include: '**/dependencies.csv' )
+    headerLine = "name,version,url,license"
+    target = new File(System.getProperty('csv')?: "${project.buildDir}/reports/dependencies/es-hadoop-dependencies.csv")
```

Comment:
mabye include a usage line in the comment ? e.g.
`usage: ./gradlew :dist:generateDependenciesReport -Dcsv=/tmp/deps.csv" `